docs/install/index.md

```diff
@@ -65,31 +91,30 @@ pip install onnxruntime-gpu --extra-index-url https://aiinfra.pkgs.visualstudio.
 
 For previous versions, you can download here: [1.18.1](https://pypi.org/project/onnxruntime-gpu/1.18.1/), [1.18.0](https://pypi.org/project/onnxruntime-gpu/1.18.0/)
 
-#### Install ONNX Runtime GPU (ROCm)
-For ROCm, please follow instructions to install it at the [AMD ROCm install docs](https://rocm.docs.amd.com/projects/install-on-linux/en/docs-6.0.0/). The ROCm execution provider for ONNX Runtime is built and tested with ROCm 6.0.0. 
-
-To build from source on Linux, follow the instructions [here](https://onnxruntime.ai/docs/build/eps.html#amd-rocm).
 
-### Install ONNX to export the model
+### Install ONNX Runtime QNN
 
 ```bash
-## ONNX is built into PyTorch
-pip install torch
+pip install onnxruntime-qnn
 ```
 
-```bash
-## tensorflow
-pip install tf2onnx
-```
+#### Install nightly
 
 ```bash
-## sklearn
-pip install skl2onnx
+pip install --pre --extra-index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ onnxruntime-qnn
```

Comment:
Sometimes the doc says "--extra-index-url", sometimes it says "--index-url". Better to keep consistent unless you have a special reason. 
The two args work differently. 
"--extra-index-url" means you will search the packages from two places: pypi.org and the extra one. 
"--index-url" means only one URL will be used.

Reply:
Which one do you recommend?

Reply:
```
pip install --pre --index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ onnxruntime
```
And setup a nightly pipeline to test all such commands for all major platforms:
```
macOS
Windows x86_64
Windows x86_32
Windows ARM64
Linux x86_64
```

Reply:
See: https://github.com/pypa/pip/issues/8606

Reply:
Am I right in saying that if you use `--index-url`, you need to install all of the dependencies manually?

Reply:
Yes.  --extra-index-url is easier to use.

Reply:
So, we recommend using "--extra-index-url" ?  I think it's fine too.